backend/account/tests/test_views.py

```diff
@@ -145,6 +145,79 @@ def test_invalid_oauth_properties(self, mock_get_or_create_user):
         mock_get_or_create_user.assert_not_called()
 
 
+class PrivacyTests(TestCase):
+    def setUp(self):
+        self.client.cookies = SimpleCookie({"csrftoken": "valid_csrf_token"})
```

Comment:
I don't *think* the `csrf` cookie or header is required for this test class, those are only accessed in the `get_or_create_user` endpoint afaik